example/README.md

```diff
@@ -244,13 +252,13 @@ And then invoking the workflow to install CentOS you just unpacked
 
     curl -H "Content-Type: application/json" \
     -X POST —-data @samples/noop_body.json \
-    http://localhost:9090/api/1.1/nodes/<insertTheNodeId>/obm | python -m json.tool
+    http://localhost:9090/api/2.0/nodes/<insertTheNodeId>/obm | python -m json.tool
```

Comment:
In API 2.0, using `/nodes/:identifier/obm` to add obm for a node is using verb `PUT` rather than `POST`.

Or use the `POST /obms`